code/style.css

```diff
@@ -3,147 +3,158 @@
   border-radius: 4px 0 0 4px;
   background: #e5e9ff;
-  color: #0026ff;
+  color: brown;
   padding: 16px;
-  font-size: 16px;
-  font-family: 'Montserrat';
+  font-family: "Montserrat", sans-serif;
   font-weight: 600;
+  font-size: 16px;
   line-height: 26px;
   flex: 1;
 }
 
-main {
-  margin: 0 auto;
-  width: 100%;
-  max-width: 700px;
-  height: 600px;
-  border-radius: 30px;
-  background: #fff;
-  padding: 20px 24px;
-  padding-top: 0;
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
+label {
+  display: none;
 }
 
-.chat {
-  flex: 1;
-  display: flex;
-  justify-content: flex-start;
-  overflow: scroll;
-  flex-direction: column;
-  padding-bottom: 16px;
+button {
+  background: brown;
+  color: white;
+  border: none;
+  border-radius: 10px;
+  padding: 10px 30px;
+  font-family: "Montserrat", sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 26px;
+  margin: 0;
+  cursor: pointer;
 }
 
-.bot-msg {
+button:hover {
+  background-color: rgb(252, 122, 71);
+}
+
+.user-msg {
   display: flex;
-  margin: 16px 8px 0 0;
   flex-shrink: 0;
+  margin: 16px 0 10px 0;
+  justify-content: flex-end; 
 }
 
-.user-msg {
+.bot-msg {
   display: flex;
-  justify-content: flex-end;
-  margin: 16px 0 0 8px;
   flex-shrink: 0;
+  margin: 16px 0 10px 0;
 }
 
-.bot-msg img,
-.user-msg img {
+.user-msg img,
+.bot-msg img {
   width: 60px;
   height: 60px;
 }
 
+
 .bubble {
   background: #e5e9ff;
+  max-width: 40%;
+  padding: 16px 24px;
+  background: #ee6b6e;
   font-weight: 600;
   font-size: 16px;
   line-height: 26px;
-  padding: 16px 24px;
-  color: #0026ff;
-  max-width: 40%;
 }
 
-.bot-bubble {
-  border-radius: 0px 26px 26px 26px;
-  margin-left: 8px;
+.user-bubble {
+  border-radius: 24px 0 24px 24px;
+  margin-right: 10px;
 }
 
-.user-bubble {
-  border-radius: 26px 0 26px 26px;
-  margin-right: 8px;
+.bot-bubble {
+  border-radius: 0 24px 24px 24px;
+  margin-left: 10px;
 }
 
-.input-wrapper {
-  display: flex;
-  justify-content: center;
+footer p {
+  color: white;
+  font-size: 12px;
+  font-style: italic;
+  font-family: dante;
+  margin: 30px auto;
+  text-align: center;
 }
 
-.input-wrapper form {
-  width: 100%;
+
+#formWrapper {
   display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
   align-items: center;
+  margin: 0 10px 0 20px;
 }
 
-label {
-  font-size: 16px;
-  font-family: 'Montserrat';
-  font-weight: 500;
-  color: #0026ff;
-  margin-right: 20px;
+#yogaBtn,
+#cookBtn,
+#englishBtn,
+#boundBook,
+#pocketBook,
+#neverMindBtn, {
+  border-radius: 20px;
+  margin: 5px;
 }
 
-button {
-  background-color: #0026ff;
-  color: white;
-  border: none;
-  border-radius: 4px;
-  padding: 16px 40px;
-  margin-right: 4px;
-  font-size: 16px;
-  line-height: 26px;
-  font-family: 'Montserrat';
-  font-weight: 500;
-  cursor: pointer;
-  transition: all 0.3s ease;
-}
-
-button:hover {
-  opacity: 0.9;
-  transition: all 0.2s ease;
+h3 {
+  color: #ee6b6e;
+  font-family: dante;
 }
```

Comment:
Maybe you could have some media query here, to make your website more responsive?